sdk/core/azure-core/azure/core/async_paging.py

```diff
@@ -85,10 +86,10 @@ def __init__(
         self._extract_data = extract_data
         self.continuation_token = continuation_token
         self._did_a_call_already = False
-        self._response = None
-        self._current_page = None
+        self._response: Optional[ResponseType] = None
```

Comment:
Given we already give the default value, do we need "Optional" here?

Reply:
Yes, Optional means "this can be None", which is the case since the default is None. IOW, any fields where the default is None is by design Optional